src/core/annotation.js

```diff
@@ -1076,6 +1076,7 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
         break;
       }
     }
+    this.checkedAppearance = normalAppearanceState.get(this.data.buttonValue);
```

Comment:
Once the checkbox PR is merged, this most likely also requires setting `uncheckedAppearance`.